lib/src/main/java/graphql/nadel/engine/transform/partition/NadelPartitionTransformHook.kt

```diff
@@ -30,17 +30,13 @@ interface NadelPartitionTransformHook {
         service: Service,
         overallField: ExecutableNormalizedField,
         hydrationDetails: ServiceExecutionHydrationDetails?,
-    ): NadelFieldPartitionContext {
```

Comment:
No point in having defaults. Those functions are necessary for partitioning to work at all